aerospike_nagios.py

```diff
@@ -600,12 +652,18 @@ def is_outside(value, start, end):
             args.stat = ">" + args.stat
         if value != None:
             stat_line = 'Aerospike Stats - ' + arg_value + ": " + args.stat + "=" + value
+elif args.set:
+    value = search(r, args.stat, delim=':')
+    if value != None:
+        stat_line = 'Aerospike Stats - ' + args.stat + "=" + value
+elif args.bin:
+    value = search(r, args.stat, delim=',')
+    if value != None:
+        stat_line = 'Aerospike Stats - ' + args.stat + "=" + value
 else:
-    for s in r.split()[-1].split(";"):    # remove leading category, then split k=v tuples
-        if s.startswith(args.stat + "="):
-            value = s.split(args.stat + "=")[-1]
-        if value != None:
-            stat_line = 'Aerospike Stats - ' + args.stat + "=" + value
+    value = search(r, args.stat)
+    if value != None:
+        stat_line = 'Aerospike Stats - ' + args.stat + "=" + value
```

Comment:
I know this pattern was in the old code but the comparisons with `None`, `value != None` should use `value is not None` since `None` is a singleton. https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
Alright, this must be because there is added functionality in "is not" that there is not in "!=" that handles the case of a singleton?

Reply:
It's a Python convention. `!=` checks inequality via the inverse of the left objects `__eq__()` or the `__ne__()` method which can be inefficient  and can cause unexpected behavior if the object has implemented an `__eq/ne__()` method that has special handling for comparisions with `None`. `is not None` is a faster identity check. It checks to see if the two objects point to the same thing (or not the same thing because of `not`). Since `None` is a singleton and there will only ever be one instance of `None`, all variables assigned to `None` reference the same object so you can always check for `None` using `is`.